app/javascript/chat/actions.js

```diff
@@ -1,3 +1,5 @@
+import { fetchSearchAPIPromise } from '../src/utils/search';
```

Comment:
can I suggest a slightly different name? `fetchSearch` ?

- the suffix `Promise` seems to be redundant, as we don't tend to add the data type as part of the variable. Imagine if all variables were named `somethingString`, `somethingArray` and so on.

- `API` is a little bit confusing to me as this is not part of the part of the DEV API, though technically `/search` is still an API, well... you get what i mean :D

`fetchSearch` seems just enough

I see you also added the documentation, which helps to understand what this method requires as arguments and what it returns

Reply:
I agree 💯% with @rhymes's suggestion here.

Reply:
Agreed on the `API` bit, I'll remove that.

I see your point about the suffix `Promise`. I originally didn't do it because usually in a helper performing an API call, I'd naturally expect a response object vs a `Promise` that I have to continue to handle.

Sounds like that's more a "me" thing with that sort of expectation and it's now documented that it returns a `Promise`, so I'm good to update it to `fetchSearch` 👍